product_docs/docs/pem/9/monitoring_performance/probes.mdx

```diff
@@ -165,13 +165,18 @@ Use the **General** tab to modify the definition of an existing probe or to spec
 
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
--   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI. Configure the **Target Type** in probes to ensure that the collected metrics are relevant and accurate for the intended level of the system hierarchy. With the **Target Type**, you can effectively monitor various components, ranging from individual database objects like tables and indexes, up to server-wide monitoring.
+-   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI. Configure **Target Type** in probes to ensure that the collected metrics are relevant and accurate for the intended level of the system hierarchy. By setting the target type, you can effectively monitor various components, ranging from individual database objects like tables and indexes to server-wide monitoring.
 
-    <details><summary>Target Type configuration matrix</summary>
+    <details><summary>Target type configuration matrix</summary>
 
-    The following table provides an overview of **Target Type** configurations, their monitoring scope, and the mandatory fields you must provide when configuring them.
+    The following table provides an overview of the target type configurations, their monitoring scope, and the mandatory fields for configuring them.
 
-    In this table, the target type column determines the object you want to monitor with the probe. The execution level column determines if the monitoring probes executes once per-database, per-server, per-agent, or per-extension level. The mandatory column indicates the coloumns you must configure in the probe query to ensure the required data is collected. Finally, the probe examples column provides some existing probes you can explore to better understand how probes are used in practice.
+    In this table: 
+    
+    - The **Target type** column determines the object you want to monitor with the probe. 
```

Comment:
@nidhibhammar I normally wouldn't bold a column name, but where it's two words and both aren't capitalized, I think it's just a bit clearer this way. We don't really have a convention for this but I think the bold is as good as any.